app/helpers/task_view_helper.rb

```diff
@@ -2,7 +2,7 @@ module TaskViewHelper
   def task_view_header(choice)
     case choice&.task_view_group
     when 1 then 'Deferred offers: review and confirm'
-    when 2 then 'Deadline approaching: respond to candidate'
+    when 2 then 'Reject by Default date approaching'
```

Comment:
Is **Default** meant to be upcase or was it written like that in error?

Reply:
@despo I wondered about this too, I've assumed this should be presented as written in Trello, @adamsilver @johndoates can you confirm?